README.md

```
@@ -104,3 +138,9 @@ npm run test:integration # 統合テスト
 ## ライセンス
 
 MIT License
+
+
+#### Quick Capture設定
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)